test-requirements.txt

```diff
@@ -1 +1,2 @@
 odoo_test_helper
+odoo-addon-mis_builder @ git+https://github.com/OCA/mis-builder.git@refs/pull/607/head#subdirectory=mis_builder
```

Comment:
esto es correcto?

Reply:
Sí, depende de esta pr.

Reply:
lo digo por el error en tests:
Run for reqfile in requirements.txt test-requirements.txt ; do
odoo-addon-mis_builder @ git+https://github.com/OCA/mis-builder.git@refs/pull/607/head#subdirectory=mis_builder
Unreleased dependencies found in test-requirements.txt.
Error: Process completed with exit code 1.

Reply:
Ángel, echa un vistazo a esta referencia para entenderlo: 

https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29